beacon_node/lighthouse_network/src/rpc/handler.rs

```diff
@@ -33,7 +33,10 @@ use tokio_util::time::{delay_queue, DelayQueue};
 use types::{EthSpec, ForkContext};
 
 /// The time (in seconds) before a substream that is awaiting a response from the user times out.
-pub const RESPONSE_TIMEOUT: u64 = 10;
+pub const RESP_TIMEOUT: u64 = 10;
+
+/// Time to first byte of a response before the request times out.
+const TTFB_TIMEOUT: u64 = 5;
```

Comment:
We could also import this constant from `protocol.rs` instead of redefining it here.